sections/what.tex

```diff
@@ -18,7 +18,9 @@ \section*{What is a hackathon?}
 For example, the astronomy community is relatively small and has a foundation of shared approaches and software implementations, allowing for a greater focus on project work over formal tutorials.
 In contrast, both the neuro- and geoscience communities covered a broader range of sub-disciplines and had a less cohesive set of existing practices, calling for greater focus on tutorials and education.
 
+Three design considerations distinguish the hack week from the summer school model: (1) hack week transfer knowledge across many levels of senority, with participants routinely leading tutorials, in part due to the relative newness of the content and the distributed knowledge base; (2) hack weeks contain a significant amount of unstructured time, creating the conditions for improvised exchanges and project development; and (3) hack weeks intentionally priortize participant diversity because this is known to maximize the liklihood for the co-creation of learning during the event.  
```

Comment:
Thinking about this and re-reading the reviewer's comments, I'm wondering if we need to be even more explicit here.

I think summer schools generally teach "established knowledge", usually taught by senior people who have worked on whatever they're teaching for years or decades. 
Hack weeks, to my mind, prioritize much more recent advances, interdisciplinary approaches and stuff that isn't "accepted canon" yet. Hack weeks encourage learning by *experimentation* rather than via lectures, exercises or even well-defined projects with a mentor. I think we kind of do say all that, but maybe not quite clear enough. 

It might also be worth being more explicit about the *goals* of various events. Summer schools mostly aim to educate junior researchers in a particular subfield in order to prepare them for a research career. Because most participants will be on the same junior level and focused much more on education, one is much less likely to find new collaborators or actively network at a summer school than one is at a hack week, where an undergraduate might work with a postdoc on a hack, or a graduate student with a senior professor.

Industry hackathons usually don't have much of an education component: they're usually focused on finding a creative solution to a specific problem or question, in a short amount of time, often phrased as a competition.

I think the closest model is probably the "unconference", which isn't really focused on education, either, and tend to be more "programmatic" than practical. Unconference sessions usually seem to be discussions that might come up with specific actionable ideas, but aren't very focused on actively creating something tangible. 

Hack weeks try to incorporate many of these approaches and ideas at the same time (and if you think about it, it's kind of astonishing how well they seem to work at doing that).